wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ParticipantsStringFormatter.swift

```diff
@@ -183,6 +183,14 @@ final class ParticipantsStringFormatter {
         case .started(withName: .some):
             result = "\(Key.with.localized) \(nameSequence.string)" && font && textColor
 
+        case .removedAnonymously:
+            var formatPath = "plural"
+            if names.names.count == 1 {
+                formatPath = isSelfIncludedInUsers ? "you" : "singular"
+            }
+            let formatString = "content.system.conversation.removed.anonymously.\(formatPath)"
+            result = formatString.localized(args: nameSequence.string) && font && textColor
```

Comment:
any chance we can use the concrete types from `Strings+Generated` instead of raw strings?